### ISSUE TITLE
Do you use detection or track bbox for proposal generation?

### ISSUE THREAD

**Issue by wenyuqing** [megvii-research/MOTRv2#34]
Hi, I'm a little confused how to generate det_bbox.json for Dancetrack? 
The first question is what is the yolox weight? Is it obtained by pretraining Bytetrack on Dancetrack dataset?  Do you use other datasets to train Bytetrack?
The second question is do you use the detection bbox of yolox or use the track bbox of Bytetrack? I am wondering whether track bbox will be better?

**Comment by zyayoung** [comment]
0. https://github.com/zyayoung/ByteTrackInference
1. The yolox weight is from https://github.com/noahcao/OC_SORT/blob/master/docs/MODEL_ZOO.md
2. we use the detection bbox of yolox for training and inference.

**Comment by wenyuqing** [comment]
Thanks a lot for your quick reply. I'm still a little bit confused about the yolox weights. Which model exactly is used and have you tried other yolox weights?
I previously thought that the yolox weight is from training yolox with ByteTrack tracker on DanceTrack training set, which according to the results in this link you provided in the paper: [https://github.com/DanceTrack/DanceTrack] as you provided in the paper ( 47.7 HOTA)
![image](https://user-images.githubusercontent.com/48059861/227089891-5039f662-cd69-4586-b410-d1c90408e20b.png)

But it seems that you use OC-SORT tracker rather than ByteTrack tracker?  Do you mean you use the model that have 55.1 HOTA results on DanceTrack test set?
![image](https://user-images.githubusercontent.com/48059861/227090295-270da165-9e46-4ed3-aa50-f3614c8a9b24.png)

Have you tried other yolox weights such as the yolox pretrained with ByterTrack tracker(47.7 HOTA)? Will it influence the overall performance a lot?

Looking forward to your reply, thanks!
